src/install/local/automated-install.md

```diff
@@ -0,0 +1,42 @@
+# Automated installation
+----
+
+Our automated installation process is setting up all the required dependencies like
+- Docker
+- Kind cluster
+- Npm
+- Aws CDK
+- Go
+- Helm (we support [Helm v3.13.0](https://github.com/helm/helm/releases/tag/v3.13.0) and later)
+
+Here are installation steps:
+
+- Git Clone to a local folder [MDAI infrastructure installation repo](https://github.com/DecisiveAI/mdai-inkops)
+- Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) for local cluster management using docker containers
+   - if you have brew installed: `brew install kind`
+- Run automated installation script
+```bash
```

Comment:
`shell`, not bash.. cleanup from last release

Reply:
resolved. adding to next commit.